### PR TITLE
Fix: Align -n command argument

### PR DESCRIPTION
**Pull request by chivalryq**
### Description of your changes

Align commands to use `-n` means namespace

Fixes #

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Review by Somefive — APPROVED** [review]
lgtm

File: references/cli/def.go [hunk 1]
```diff
@@ -47,22 +47,6 @@ import (
...
-	// FlagOutput command flag to specify which file to save
-	FlagOutput = "output"
-	// FlagMessage command flag to specify which file to save
-	FlagMessage = "message"
-	// FlagType command flag to specify which definition type to use
-	FlagType = "type"
-	// FlagNamespace command flag to specify which namespace to use
-	FlagNamespace = "namespace"
-	// FlagInteractive command flag to specify the use of interactive process
-	FlagInteractive = "interactive"
```
Comment (wonderflow) [inline references/cli/def.go:65]: why they're deleted?

File: references/appfile/api/appfile.go [hunk 2]
```diff
@@ -142,7 +141,7 @@ func (app *AppFile) ExecuteAppfileTasks(io cmdutil.IOStreams) error {
 }
 
 // BuildOAMApplication renders Appfile into Application, Scopes and other K8s Resources.
-func (app *AppFile) BuildOAMApplication(env *types.EnvMeta, io cmdutil.IOStreams, tm template.Manager, silence bool) (*v1beta1.Application, []oam.Object, error) {
```
Comment (wonderflow) [inline references/appfile/api/appfile.go:145]: we should keep the env concept and add `namespace` args as override. Keep the compatibility

Comment (chivalryq) [reply references/appfile/api/appfile.go:145]: I think we shouldn't keep env concept any more. There are three reasons 
1. It's originally for create namespace. If we keep 2 concept, users will be confused.
2. As we all focus on release 1.2 now. There are already a lot concepts about envrionment. We shouldn't keep this as another environment.
3. It's easy to swtch from -e to -n.

Comment (wonderflow) [reply references/appfile/api/appfile.go:145]: 如果要删就要整体都删，不能半吊子

Comment (chivalryq) [reply references/appfile/api/appfile.go:145]: 我遗留的部分是exec/port-forward/logs，在 @Somefive 的工作流程中，他合并后我会继续把这个概念清理干净

Comment (wonderflow) [reply references/appfile/api/appfile.go:145]: 那先把 vela env 命令清理掉

**Review by wonderflow — CHANGES_REQUESTED** [review]
we should keep the compatibility with `vela env` until we have alternative plan

**Review by wonderflow — APPROVED** [review]
good job

**Pull request merged**